sycl/doc/GetStartedGuide.md

```diff
@@ -55,55 +59,49 @@ Open a developer command prompt using one of two methods:
 ```bat
 set DPCPP_HOME=%USERPROFILE%\sycl_workspace
 mkdir %DPCPP_HOME%
+cd %DPCPP_HOME%
+
+git clone https://github.com/intel/llvm -b sycl
```

Comment:
I'm not sure if we really want to do this. But shallow clone would dramatically decrease download times for those who just want's to build the latest version (instead of working on new features/bugfixes).
```suggestion
git clone --depth 1 https://github.com/intel/llvm -b sycl
```